ledger/src/bigtable_upload.rs

```diff
@@ -18,6 +18,9 @@ use {
 // Attempt to upload this many blocks in parallel
 const NUM_BLOCKS_TO_UPLOAD_IN_PARALLEL: usize = 32;
```

Comment:
note: i tried increasing this to 64 (running on 48 thread cpu) and it would lockup. don't really have an idea why this was happening yet. suspicious on the tokio runtime/something locking it up

Reply:
with that many threads the locking could just be resource contention with the rest of the validato. my understanding is this upload service was originally supposed to run as just one background thread so would have a low resource impact